shared/src/main/scala/purecsv/safe/converter/Converter.scala

```diff
@@ -45,7 +48,7 @@ object StringConverter {
 
 object StringConverterUtils {
   def mkStringConverter[A](fromF: String => Try[A], toF: A => String) = new StringConverter[A] {
-    def tryFrom(s: String): Try[A] = fromF(s)
+    def tryFrom(s: String, trimming: Trimming): Try[A] = fromF(s)
```

Comment:
to not break the API, could we default this to Trimming.NoAction?

Reply:
we don't appear to pass trimming to fromF function

Reply:
Trimming has been defaulted to NoAction. No it's being passed to fromF + I've added more tests.